src/screens/Auth/LoginScreen.tsx

```diff
@@ -84,30 +88,53 @@ export const LoginScreen = (props: LoginScreenProps) => {
     );
 
     //using the loadstates, determines if loading page should be shown
-    const loadState: loadStateActionTypes = determineLoadState({
+    const loadState: LoadStateActionTypes = determineLoadState({
         AuthState,
         ProfileState,
         TasksState,
     });
     //if everything is loaded, change state to login
-    if (loadState === loadStateActionTypes.LOADED) dispatch(login());
+    if (loadState === LoadStateActionTypes.LOADED) dispatch(login());
 
     //error message handling
     let errorMessage: React.ReactNode | null = null;
-    if (loadState === loadStateActionTypes.ERROR) {
+    if (loadState === LoadStateActionTypes.ERROR) {
         //can be cleaned up better, any suggestions?
         if (AuthError) errorMessage = createErrorMessage(AuthError.message);
         else if (ProfileError) errorMessage = createErrorMessage(ProfileError.message);
         else if (TasksError) errorMessage = createErrorMessage(TasksError.message);
         else errorMessage = null;
     }
 
-    if (loadState === loadStateActionTypes.LOADING) return <LoadingPage opacity={0.8} />;
-
     const clearInputHandler = (inputField) => {
         if (inputField === 'email') return setEmail({ ...email, value: '' });
     };
 
+    //will be used to handle receiving of notifications once activity feed is finished
+    useEffect(() => {
+        //run when notification is received and tapped and when app is NOT running
+        const backgroundSubscription = Notifications.addNotificationResponseReceivedListener(
+            (response) => {
+                // console.log('response: ', response);
+                Notifications.setBadgeCountAsync(0);
+            },
+        );
+
+        //run when notification is received and when app is running
+        const foregroundSubscription = Notifications.addNotificationReceivedListener(
+            (notification) => {
+                // console.log('notification', notification);
+            },
+        );
+
+        return () => {
```

Comment:
https://reactjs.org/docs/hooks-effect.html
the return function in useeffect() is run when the component unmounts, so calling .remove() on the subscriptions will clean it up. This useffect will be rewritten, I left it here as an example of how to add subscriptions to notification interactions

Reply:
what does adding subscriptions to notification interactions mean? also my question is more about how do you know the component will be unmounted?

Reply:
if your comment is also to answer the subscription that doesn't do anything, you can just comment it out but keep it? So no one's confused at why it exists, at least if it's commented it can be kept as mental note for the future

Reply:
ok i commented it out. 
This is in the login screen, so its unmounted when login is successful. 
Subscriptions just means what happens when a notification is tapped on